tests/suites/test_suite_ssl.function

```diff
@@ -17,6 +17,9 @@
 #define TEST_EARLY_DATA_NO_INDICATION_SENT 1
 #define TEST_EARLY_DATA_SERVER_REJECTS 2
 #define TEST_EARLY_DATA_HRR 3
+#define TEST_EARLY_DATA_SAME_ALPN 4
```

Comment:
```suggestion
#define TEST_EARLY_DATA_SAME_ALPN 4
```
I'd suggest to remove that scenario and just define some ALPNs if MBEDTLS_SSL_ALPN is enabled as part of TEST_EARLY_DATA_ACCEPTED.

Reply:
Updated, I suggest to remove it actually.

Reply:
I added it so that we can test if no ALPN is negotiated we don't do the check.